content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -363,6 +363,122 @@ to configure the `service` tag only in the configuration of the process check.
 {{% /tab %}}
 {{< /tabs >}}
 
+### Serverless environment
+
+#### AWS Lambda Functions 
+
+Depending on how you build and deploy your AWS Lambda-based serverless applications, you may have several options available for applying the `env`, `service` and `version` tags to metrics, traces and logs.
+
+Note, these tags are expected to be specified through AWS resource tags, instead of environment variables. Specifically, the `DD_ENV`, `DD_SERVICE` and `DD_VERSION` environment variables are not supported.
```

Comment:
```suggestion
*Note*: These tags are specified through AWS resource tags instead of environment variables. Specifically, the `DD_ENV`, `DD_SERVICE` and `DD_VERSION` environment variables are not supported.
```